_data/training-modules.yaml

```
@@ -1,3 +1,17 @@
+- description: 'Learn about ROOT, RooFit, machine learning with TMVA, and physics simulations.'
+  webpage: https://gitlab.com/stroche/particle-physics-methods
+  name: "Particle physics methods"
+  repository: https://gitlab.com/stroche/particle-physics-methods
+  id: root-analysis
+  videos: ''
+  status: beta
+- description: 'This course covers various best practices like testing, pytest, object oriented programming, packing, CI, and more.'
+  name: "Software Engineering for Scientific Computing"
+  repository: https://github.com/henryiii/se-for-sci
+  status: stable
+  videos: ''
+  webpage: https://henryiii.github.io/se-for-sci/
+  id: se-for-sci
 - description: 'Track code changes, undo mistakes, collaborate. This module is a must.'
```

```suggestion
 - description: 'Track code changes, undo mistakes, collaborate. This module is a must.'
```
This is more than a module and overlaps with many of our other modules. I'd avoid calling it a 'must' unless you specify something like 'for project maintainers' or so.

---

That's the description for the next one :)

---

(I think a yaml formatter originally sorted the keys alphabetically)

---

Haha, I see now.